visionSamples/barcode-reader/app/src/main/java/com/google/android/gms/samples/vision/barcodereader/BarcodeTrackerFactory.java

```diff
@@ -24,17 +26,22 @@
  * Factory for creating a tracker and associated graphic to be associated with a new barcode.  The
  * multi-processor uses this factory to create barcode trackers as needed -- one for each barcode.
  */
-class BarcodeTrackerFactory implements MultiProcessor.Factory<Barcode> {
-    private GraphicOverlay<BarcodeGraphic> mGraphicOverlay;
+public class BarcodeTrackerFactory implements MultiProcessor.Factory<Barcode> {
+    private GraphicOverlay<BarcodeGraphic> graphicOverlay;
+    Context context;
 
-    BarcodeTrackerFactory(GraphicOverlay<BarcodeGraphic> barcodeGraphicOverlay) {
-        mGraphicOverlay = barcodeGraphicOverlay;
+    public BarcodeTrackerFactory(GraphicOverlay<BarcodeGraphic> graphicOverlay,
+                                 Context context) {
+        this.graphicOverlay = graphicOverlay;
+        this.context = context;
     }
 
     @Override
     public Tracker<Barcode> create(Barcode barcode) {
-        BarcodeGraphic graphic = new BarcodeGraphic(mGraphicOverlay);
-        return new BarcodeGraphicTracker(mGraphicOverlay, graphic);
+
```

Comment:
Please remove this blank line and the blank line at the end of the method.

Reply:
Done.